README.md

```diff
@@ -22,7 +22,8 @@ The game implements the following (and more):
 - Voxel explosions
 - Reading game maps from PNG image files
 - ".vox" file loading into chunks
-- etc.. 
+- etc.....
```

Comment:
From my understanding, two dots is an informal and cute way of implying there is something else to say, except philosophically you don't need to say it at that moment, so it's more of a short-cut, and yes two dots is quicker that three since the latter means something else, formally any-way.

However, I'm unaware of the meaning behind five dots.

Reply:
Prison inmates or former inmates will also brand themselves with a 5-dot tattoo, or the “quincunx.” Four dots represents the four walls of a prison, while the fifth signifies the person inside.